src/main/java/calculator/Delimiter.java

```diff
@@ -0,0 +1,33 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Delimiter {
+
+    private static String basicDelimiter = ",:";
```

Comment:
final 을 붙여서 상수화 할 수 있을 것 같아요.

추가로 상수는 보통 아래와 같이 작성합니다.

```suggestion
    private static final String BASIC_DELIMITER = ",:";
```